clifford/tools/g3c/__init__.py

```diff
@@ -813,42 +813,45 @@ def get_nearest_plane_point(plane):
     return up(get_plane_normal(plane)*get_plane_origin_distance(plane))
 
 
-def disturb_object(mv_object, maximum_translation=0.01, maximum_angle=0.01):
+def disturb_object(mv_object, maximum_translation=0.01, maximum_angle=0.01, *, rng=None):
     """ Disturbs an object by a random rotor """
-    r = random_rotation_translation_rotor(maximum_translation=maximum_translation, maximum_angle=maximum_angle)
+    r = random_rotation_translation_rotor(maximum_translation=maximum_translation,
+                                          maximum_angle=maximum_angle, rng=rng)
     return (r*mv_object*~r).normal()
 
 
-def generate_n_clusters(object_generator, n_clusters, n_objects_per_cluster):
+def generate_n_clusters(object_generator, n_clusters, n_objects_per_cluster, *, rng=None):
     """ Creates n_clusters of random objects """
     object_clusters = []
```

Comment:
since you use it in a loop:
```suggestion
    rng = np.random.default_rng(rng)
    object_clusters = []
```